kpi/utils/object_permission_helper.py

```diff
@@ -32,7 +32,12 @@ def get_user_permission_assignments_queryset(cls, affected_object, user):
         # Filtering is done in `get_queryset` instead of FilteredBackend class
         # because it's specific to `ObjectPermission`.
         if not user or user.is_anonymous:
-            queryset = queryset.filter(user_id=affected_object.owner_id)
+            queryset = queryset.filter(
+                user_id__in=[
+                    affected_object.owner_id,
+                    settings.ANONYMOUS_USER_ID,
+                ]
+            )
```

Comment:
Great... Because the code is almost the same I would have refactored it a little bit. 

```python3
        # Filtering is done in `get_queryset` instead of FilteredBackend class
        # because it's specific to `ObjectPermission`.
        if not affected_object.has_perm(user, PERM_MANAGE_ASSET):
            filtered_users = [
                affected_object.owner_id,
                settings.ANONYMOUS_USER_ID,
            ]

            # Display only users' permissions if they are not allowed to modify
            # others' permissions
            if not user.is_anonymous:
                filtered_users.append(user.pk)

            queryset = queryset.filter(user_id__in=filtered_users)
        
        return queryset
```
The difference in my code, it does not handle `user is None`. But I looked at the code and `get_user_permission_assignments_queryset` is never called without `user`